### ISSUE TITLE
specifying service type right now is very docker-compose specific

### ISSUE THREAD

**Issue by surajssd** [kubernetes-incubator/kompose#273]
We use `labels` from docker-compose to specify type of service (e.g. `kompose.service.type: nodeport`) that gets created and this gets mapped to [`annotatations`](https://github.com/kubernetes-incubator/kompose/blob/a3495b1d6bf73dba0fd6f3af44d68f2f910e3df0/pkg/kobject/kobject.go#L135) internally but then we can do this cleanly by adding one more directive in [`ServiceConfig`](https://github.com/kubernetes-incubator/kompose/blob/a3495b1d6bf73dba0fd6f3af44d68f2f910e3df0/pkg/kobject/kobject.go#L124) named `ServiceType string` so that at [loader stage](https://github.com/kubernetes-incubator/kompose/blob/a3495b1d6bf73dba0fd6f3af44d68f2f910e3df0/pkg/loader/compose/compose.go#L127) itself we load this variable and [in transformation](https://github.com/kubernetes-incubator/kompose/blob/master/pkg/transformer/kubernetes/k8sutils.go#L247) we don't need to do annotation checking!

So basically now we add one vairable to `ServiceConfig` and move code from [`CreateService`](https://github.com/kubernetes-incubator/kompose/blob/master/pkg/transformer/kubernetes/k8sutils.go#L239) to respective loaders.

**Comment by ngtuna** [comment]
+1. I think that makes sense.
